curriculum/challenges/english/14-responsive-web-design-22/build-a-tribute-page-project/build-a-tribute-page.md

```diff
@@ -12,7 +12,7 @@ dashedName: build-a-tribute-page
 
 **User Stories:**
 
-1. Your tribute page should have an element with a corresponding `id="main"`, which contains all other elements
+1. Your tribute page should have a `main` element with a corresponding `id` of `"main"`, which contains all other elements
```

Comment:
I don't think main should be wrapped in quotes here
`id` of `"main"`

If you look at steps 2&3, there are no quotes
`id` of `title`
`id` of `img-div`

Quotes were only used in the context of the attribute 
`id="main"`

Proposed fix: `id` of `main`

Reply:
Makes sense, thank you for the additional idea. I'll do the change right away.